test/integration/Marketplaceflow.test.ts

```diff
@@ -561,12 +561,12 @@ describe('Marketplace flow', () => {
   })
   it('Bob tries to consumes asset with bad URL, but tokens are not deducted', async () => {
     const balance = await datatoken.balance(tokenAddressForBadUrlAsset, bob.getId())
-    const txid = await ocean.assets.order(
-      ddoWithBadUrl.id,
-      accessService.type,
-      bob.getId()
-    )
-    assert(txid === null)
+    try {
+      await ocean.assets.order(ddoWithBadUrl.id, accessService.type, bob.getId())
+    } catch (error) {
+      assert(error != null)
```

Comment:
if order succeeds, your catch block is not going to be executed, so you will have no assertions failed.

Reply:
good point 👍  thanks, going to update this